logfire/__init__.py

```diff
@@ -150,9 +151,11 @@ def loguru_handler() -> Any:
     'ScrubbingOptions',
     'VERSION',
     'suppress_instrumentation',
+    'SamplingOptions',
+    'MetricsOptions',
+    # Integrations
     'StructlogProcessor',
     'LogfireLoggingHandler',
     'loguru_handler',
-    'SamplingOptions',
-    'MetricsOptions',
+    'CeleryInstrumentKwargs',
```

Comment:
I think we should in general avoid adding things to `__init__` if they only serve as type hints.